.github/tests/lm_tests.py

```diff
@@ -167,6 +167,34 @@ def test_group_by_with_agg(setup_models, model):
     assert set(cleaned_df["final_output"].values[1].lower().strip(".,!?\"'").split(", ")) == {"michael", "dwight"}
 
 
+@pytest.mark.parametrize("model", get_enabled("gpt-4o-mini"))
+def test_sem_extract(setup_models, model):
+    lm = setup_models[model]
+    lotus.settings.configure(lm=lm)
+
+    data = {
+        "Text": [
+            "Lionel Messi is a good soccer player, he has won the World Cup 5 times",
+            "Michael Jordan is a good basketball player, he has won the NBA championships 6 times",
+            "Tiger Woods is a good golf player, he has won the Master championships 4 times",
+            "Tom Brady is a good football player, he has won the NFL championships 7 times",
+        ]
+    }
+    df = pd.DataFrame(data)
+    user_instruction = "{Text}"
+    columns = ["Name", "Sport", "Number of Championships"]
+    df = df.sem_extract(columns=columns, user_instruction=user_instruction)
+
```

Comment:
Let's also add back in the == checks against `Name`, `Sport`, `Number of Championships`

Reply:
wdym, we want to check if the column names match the columns?

Reply:
Check that `Lionel Messi`, `soccer`, `5` are the actual row contents

Reply:
You should also just do the check in lowercase and strip whitespace so its not flaky